src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.csproj

```diff
@@ -489,10 +489,11 @@
     <Compile Include="Bindings\BindingProviders\Rule.cs" />
     <Compile Include="Bindings\BindingProviders\FluentBindingProvider.cs" />
     <Compile Include="Bindings\BindingProviders\FilteringBindingProvider.cs" />
-    <Compile Include="Bindings\BindingProviders\FluidBindingProviderExtensions.cs" />
+    <Compile Include="Bindings\BindingProviders\FluentBindingProviderExtensions.cs" />
```

Comment:
Note that this change will break the Bot binding I believe since they're currently doing reflection. You'll have to stage this right :)